test/snippet/alphabet/quality/qualified.cpp

```diff
@@ -3,32 +3,30 @@
 #include <seqan3/alphabet/nucleotide/dna4.hpp>
 #include <seqan3/core/debug_stream.hpp>
 
-using namespace seqan3;
-
 int main()
 {
-//! [general]
-qualified<dna4, phred42> letter{'A'_dna4, phred42{7}};
-debug_stream << int(to_rank(letter)) << ' '
-          << int(to_rank(get<0>(letter))) << ' '
-          << int(to_rank(get<1>(letter))) << '\n';
-// 28 0 7
+    using seqan3::operator""_dna4;
+    using seqan3::get;
+
+    seqan3::qualified<seqan3::dna4, seqan3::phred42> letter{'A'_dna4, seqan3::phred42{7}};
+    seqan3::debug_stream << int(seqan3::to_rank(letter)) << ' '
+                         << int(seqan3::to_rank(get<0>(letter))) << ' '
+                         << int(seqan3::to_rank(get<1>(letter))) << '\n';
```

Comment:
the cast to int is not required anymore, the debug_stream prints small numbers as numbers and not as chars.